index.html

```diff
@@ -61,8 +61,231 @@ <h1 class="box1">
+                   <div class="project_info">
+                     <p class="project_title_II">
+                      CANOPY
+                     </p>
+                     <ul class="project_job_title_info_container">
+                         <li class="job_title_info">
+                           Back End Dev
+                         </li>
+                         <li class="job_title_info">
+                           2015
+                         </li>
+                     </ul>
+                   </div>
+                   <p class="project_full_desc">
+                     A daily selection of privately personalized reads; no accounts or sign-ups required.
+                   </p>
+                   <div class="project_tags_container">
+                       <p class="project_tag">html</p>
+                       <p class="project_tag">css</p>
+                       <p class="project_tag">javaScript</p>
+                   </div>
+                   <button type="button" class="view_project_btn">
+                       See Project
+                   </button>
+                </div>
+              </li>
+             </ul>
+            </section>
 
+            <section class="about_container">
+              <h1 class="box1">
+                  About Myself
+              </h1>
+              <p class="box2">
+                Hello I'm a software developer! I can help you build a product, 
+                feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, don't hestiate to contact me.
+              </p>
+              <div class="box3">
+                  LET'S CONNECT
+              </div>
+              <ul class="box4">
+                   <li class="socials">
+                      <a href="https://twitter/harlexkhal">
+                        <img src="images/twitter.png" alt="twitter">
+                       </a>
+                   </li>
+                   <li class="socials">
+                     <a href="https://www.linkedin.com/in/alexander-oguzie-ibeh-776814164">
+                       <img src="images/Linkedin.png" alt="Linkedin">
+                     </a>
+                   </li>
+                   <li class="socials child">
+                     <a href="#">
+                       <img src="images/m.png" alt="M">
+                     </a>
+                   </li>
+                   <li class="socials">
+                     <a href="https://github.com/harlekhal">
+                       <img src="images/github.png" alt="github">
+                     </a>
+                   </li>
+                   <li class="socials">
+                     <a href="#">
+                       <img src="images/peace.png" alt="peace">
+                     </a>
+                   </li>
+              </ul>
+               <button type="button" class="resume_btn">
+                  Get my resume
+               </button>
```

Comment:
- [ ] Nice job for adding the _Pressed_ effect of this button styling. We can make it more user-friendly by adding the _Hover_ and the _Active_ effect.